deploy/crds/devconsole_v1alpha1_gitsourceanalysis_crd.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
We should really stop duplicating the same CRDs in deploy and manifest folders. We do need to store CRDs in manifests so let's try to reuse them everywhere instead of duplicating them in /deploy? **This is not for this PR to take care** though. Just saying :)

Reply:
yeah, the current way of storing the CRD yamls are so *tiring*.
Tests and everything needs to look at /deploy only for RBAC and manifests/<latest-version> for CRDs. 

Also, we need to handle the problem of rbac info staying in two places - /deploy and `clusterPermissions` in /manifests/<latest-version>

Reply:
tracked https://jira.coreos.com/browse/ODC-473

Reply:
Btw, 
FYI @baijum reminded me that there are sanity tests called as part of `make test` that verify that the CRDs are same.